src/sections/dieFounder.html

```diff
@@ -30,31 +30,35 @@ <h3 class="mt-12 mb-6 text-white font-railwayBold">
           dem Norden und Süden anbieten.
         </p>
       </div>
-      <div class="relative z-10 ml-80 mr-96">
+      <div class="relative lg:z-10 lg:ml-80 lg:mr-96 ml-6 mr-6">
         <img src="../../images/Mathias_Team.webp" alt="" />
       </div>
     </section>
     <section
-      class="relative flex flex-row text-white bg-no-repeat bg-cover -mt-[220px] h-[400px] z-0 justify-between"
+      class="lg:relative flex flex-row text-white lg:bg-no-repeat lg:bg-cover lg:-mt-[300px] lg:h-[550px] z-0 justify-between gap-4 -mt-36 md:h-[300px] md:-mt-40"
       style="background-image: url(../../images/bg_hero_main.webp)"
     >
-      <div class="mt-72 ml-72">
-        <h6 class="mb-6 text-xs font-railwayBold">Volker Maiborn:</h6>
+      <div class="lg:mt-96 lg:pl-96 mx-4 mt-36">
```

Comment:
Here on MacBook Air, you can try to make it responsive so that those paragraphs can not override the next page. 

![image](https://github.com/divinecharlotte/north-star-team7/assets/81089765/6c4b9b41-8d99-4535-b282-2fbdbd8aa0a6)